py/torch_tensorrt/dynamo/backend/__init__.py

```diff
@@ -94,6 +95,7 @@ def compile(
         workspace_size=workspace_size,
         min_block_size=min_block_size,
         torch_executed_ops=torch_executed_ops,
+        pass_through_build_failures=pass_through_build_failures,
```

Comment:
This should be in its own PR dependent on whatever provides `pass_through_build_failures`

Reply:
Split out in #1997